kratos/elements/embedded_nodal_variable_calculation_element_simplex.h

```diff
@@ -0,0 +1,432 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
+//
+//  Main authors:    Ruben Zorrilla
+//
+//
+
+#if !defined(KRATOS_EMBEDDED_NODAL_VARIABLE_CALCULATION_ELEMENT_H_INCLUDED )
+#define  KRATOS_EMBEDDED_NODAL_VARIABLE_CALCULATION_ELEMENT_H_INCLUDED
+
+// System includes
+#include <string>
+#include <iostream>
+
+
+// External includes
+
+
+// Project includes
+#include "containers/array_1d.h"
+#include "includes/define.h"
+#include "includes/element.h"
+#include "includes/serializer.h"
+#include "includes/variables.h"
```

Comment:
is this needed? => maybe move to cpp

Reply:
Probably most of them can be moved to the cpp. I'll have a look.

Reply:
Arranged.